clientdb/account.go

```diff
@@ -10,6 +10,14 @@ import (
 	"go.etcd.io/bbolt"
 )
 
+const (
+	// accountStateVersionedMask is a bit mask for detecting from the state
+	// of an account whether that account has a version field encoded with
+	// it or not. We use the first bit of the uint8 state field because we
+	// are unlikely to ever have more than 127 different states.
+	accountStateVersionedMask account.State = 0b1000_0000
```

Comment:
I fail to see why we don't use TLV fiels in a new bucket for this and future updates